src/SprykerEco/Yves/Payone/PayoneFactory.php

```diff
@@ -55,23 +65,23 @@ class PayoneFactory extends AbstractFactory
     /**
      * @return \SprykerEco\Yves\Payone\Form\PrePaymentForm
      */
-    public function createPrePaymentForm()
+    public function createPrePaymentForm(): AbstractPayoneSubForm
     {
         return new PrePaymentForm();
     }
 
     /**
      * @return \SprykerEco\Yves\Payone\Form\DataProvider\PrePaymentDataProvider
      */
-    public function createPrePaymentFormDataProvider()
+    public function createPrePaymentFormDataProvider(): PrePaymentDataProvider
     {
         return new PrePaymentDataProvider();
     }
 
     /**
      * @return \SprykerEco\Yves\Payone\Form\InvoiceSubForm
      */
-    public function createInvoiceSubForm()
+    public function createInvoiceSubForm(): InvoiceSubForm
```

Comment:
* Check the return type hints, for all changed methods.

Reply:
++